zproject/prod_settings_template.py

```diff
@@ -471,7 +471,21 @@
 # optionally using LDAP as an authentication mechanism.
 
 import ldap
-from django_auth_ldap.config import LDAPSearch
+from django_auth_ldap.config import (
+    LDAPSearch, LDAPSearchUnion, LDAPGroupQuery, PosixGroupType,
+    MemberDNGroupType, NestedMemberDNGroupType,
+    GroupOfNamesType, NestedGroupOfNamesType,
+    GroupOfUniqueNamesType, NestedGroupOfUniqueNamesType,
+    ActiveDirectoryGroupType, NestedActiveDirectoryGroupType,
+    OrganizationalRoleGroupType, NestedOrganizationalRoleGroupType
+)
+# Suppress pyflakes warnings about unused modules.
+assert (LDAPSearchUnion and LDAPGroupQuery and PosixGroupType and
+        MemberDNGroupType and NestedMemberDNGroupType and
+        GroupOfNamesType and NestedGroupOfNamesType and
+        GroupOfUniqueNamesType and NestedGroupOfUniqueNamesType and
+        ActiveDirectoryGroupType and NestedActiveDirectoryGroupType and
+        OrganizationalRoleGroupType and NestedOrganizationalRoleGroupType)
```

Comment:
So, I'm not a fan of this model, because we can't update this file on existing systems.

What do you think about this file having `from zproject.django_auth_ldap_exports import *` and then having that be a small that re-exports just these values?  I think that'd provide a lot more flexibility for updating things in the future.

Reply:
I see what you mean. I'll change the PR shortly.